index.html

```diff
@@ -565,6 +567,7 @@ <h4 class="modal-title" i18n="featureinfo.modal.title">Informations</h4>
             Sortable.create(document.getElementById('layers-container'), {
                 handle: '.layerdisplay-title',
                 draggable: '.draggable',
+                filter: '.toplayer',
```

Comment:
Permet d'être certain qu'un élément de la liste ne peut être déplacé selon la doc :
`  filter: ".ignore-elements", // Selectors that do not lead to dragging (String or Function)`

 https://github.com/SortableJS/sortablejs